ops/terraform/services/base/values/ephemeral.yaml

```diff
@@ -1,4 +1,5 @@
 ---
+# CIPHER FILE HEADER, DO NOT REMOVE (noop preamble to satisfy the pre-commit script)
```

Comment:
As discussed, I think we ought to remove both this line and the reference to `ops/terraform/services/base/values/ephemeral.yaml` from `.github/resources/.sensitive-values`

Re-iterating, with the usage of matching, environment-specific KMS keys with both in the repository via `cipher` and in the remote storage via AWS SSM, ephemeral-specific encrypted values aren't possible with the current implementation. I don't think we've identified a good use-case to support them yet, either.